src/api/app/controllers/request_controller.rb

```diff
@@ -34,10 +34,7 @@ def render_request_collection
     params[:review_states] = params[:reviewstates].split(',') if params[:reviewstates]
     params[:ids] = params[:ids].split(',').map(&:to_i) if params[:ids]
 
-    rel = BsRequest.find_for(params).includes([:reviews]).
-          includes({bs_request_actions: :bs_request_action_accept_info}).
-          order('bs_requests.id').references(:bs_requests)
-    rel = rel.limit(params[:limit].to_i) if params[:limit].to_i > 0
+    rel = BsRequest.find_for(params).includes({bs_request_actions: :bs_request_action_accept_info})
```

Comment:
I don think the curly brackets {} are necessary?

Reply:
This is just c&p but I will fix it!